core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -539,7 +539,7 @@ default Optional<ConnectorOutputMetadata> finishInsert(ConnectorSession session,
      */
     default boolean delegateMaterializedViewRefreshToConnector(ConnectorSession session, SchemaTableName viewName)
     {
-        throw new TrinoException(NOT_SUPPORTED, "This connector does not support materialized views");
+        return false;
```

Comment:
Is this needed? IMO it makes sense for a connector that supports MV to override it. `IcebergMetadata` does it already.